### DISCUSSION TITLE
Multiple ini files warning?

### DISCUSSION THREAD

**Discussion by stevespages** [composer/composer#11050] [Q&A / Support]
I ran `php composer.phar require phpmailer/phpmailer` and got a warning saying zip extension and unzip commands are both missing, skipping. Also, in the same block of yellow hightlighted text, it said: 

> Your command-line PHP is using multiple ini files. Run `php --ini` to show them.

I am assuming this is problem or potential problem as why else would it be mentioned. On the other hand, I thought it was OK for PHP to have multiple ini files. I did run `php --ini` and got the following output:

`
Configuration File (php.ini) Path: /etc/php/8.1/cli
Loaded Configuration File:         /etc/php/8.1/cli/php.ini
Scan for additional .ini files in: /etc/php/8.1/cli/conf.d
Additional .ini files parsed:      /etc/php/8.1/cli/conf.d/10-opcache.ini,
/etc/php/8.1/cli/conf.d/10-pdo.ini,
/etc/php/8.1/cli/conf.d/20-calendar.ini,
/etc/php/8.1/cli/conf.d/20-ctype.ini,
/etc/php/8.1/cli/conf.d/20-exif.ini,
/etc/php/8.1/cli/conf.d/20-ffi.ini,
/etc/php/8.1/cli/conf.d/20-fileinfo.ini,
/etc/php/8.1/cli/conf.d/20-ftp.ini,
/etc/php/8.1/cli/conf.d/20-gettext.ini,
/etc/php/8.1/cli/conf.d/20-iconv.ini,
/etc/php/8.1/cli/conf.d/20-phar.ini,
/etc/php/8.1/cli/conf.d/20-posix.ini,
/etc/php/8.1/cli/conf.d/20-readline.ini,
/etc/php/8.1/cli/conf.d/20-shmop.ini,
/etc/php/8.1/cli/conf.d/20-sockets.ini,
/etc/php/8.1/cli/conf.d/20-sysvmsg.ini,
/etc/php/8.1/cli/conf.d/20-sysvsem.ini,
/etc/php/8.1/cli/conf.d/20-sysvshm.ini,
/etc/php/8.1/cli/conf.d/20-tokenizer.ini
`

It would be great to know if having multiple ini files is a problem and if it is what the best thing to do is. I have not edited php.ini before so am new to this.

**Comment by Seldaek** [comment]
The point of this hint is to let you know how to find the ini, as if there are multiple we do not want to list all of them. So it's not a problem no, just a hint.
